deploy/dcos-node-datadog-agent.json

```diff
@@ -1,102 +1,102 @@
+      },
+      {
+        "containerPath": "/host/proc",
+        "hostPath": "/proc",
+        "mode": "RO"
+      },
+      {
+        "containerPath": "/host/sys/fs/cgroup",
+        "hostPath": "/sys/fs/cgroup",
+        "mode": "RO"
+      },
+      {
+        "containerPath": "/conf.d/mesos_slave.d",
+        "hostPath": "/opt/datadog-agent/conf.d/mesos_slave.d",
+        "mode": "RO"
+      },
+      {
+        "containerPath": "/etc/cont-init.d/90-mesos.sh",
+        "hostPath": "/opt/datadog-agent/cont-init.d/90-mesos.sh",
+        "mode": "RO"
+      }
     ],
-    "container": {
-        "type": "DOCKER",
-        "volumes": [
-            {
-                "containerPath": "/var/run/docker.sock",
-                "hostPath": "/var/run/docker.sock",
-                "mode": "RO"
-            },
-            {
-                "containerPath": "/host/proc",
-                "hostPath": "/proc",
-                "mode": "RO"
-            },
-            {
-                "containerPath": "/host/sys/fs/cgroup",
-                "hostPath": "/sys/fs/cgroup",
-                "mode": "RO"
-            },
-            {
-                "containerPath": "/conf.d/mesos_slave.d",
-                "hostPath": "/opt/datadog-agent/conf.d/mesos_slave.d",
-                "mode": "RO"
-            },
-            {
-                "containerPath": "/etc/cont-init.d/90-mesos.sh",
-                "hostPath": "/opt/datadog-agent/cont-init.d/90-mesos.sh",
-                "mode": "RO"
-            }
-        ],
-        "docker": {
-            "image": "datadog/agent-dev:dcos-auth-token-py3",
-            "network": "BRIDGE",
-            "portMappings": [
-                {
-                    "containerPort": 8125,
-                    "hostPort": 8125,
-                    "servicePort": 10000,
-                    "protocol": "udp",
-                    "labels": {}
-                }
-            ],
-            "privileged": false,
-            "parameters": [
-                {
-                    "key": "env",
-                    "value": "DD_API_KEY=<YOUR_DD_API_KEY>"
-                },
-                {
-                    "key": "env",
-                    "value": "MESOS_SLAVE=true"
-                }
-            ],
-            "forcePullImage": true
-        }
-    },
-    "healthChecks": [
+    "docker": {
+      "image": "datadog/agent-dev:dcos-auth-py3",
+      "network": "BRIDGE",
+      "portMappings": [
         {
-            "protocol": "COMMAND",
-            "command": {
-                "value": "/probe.sh"
-            },
-            "gracePeriodSeconds": 300,
-            "intervalSeconds": 60,
-            "timeoutSeconds": 20,
-            "maxConsecutiveFailures": 3
+          "containerPort": 8125,
+          "hostPort": 8125,
+          "servicePort": 10000,
+          "protocol": "udp",
+          "labels": {}
         }
-    ],
-    "portDefinitions": [
+      ],
+      "privileged": false,
```

Comment:
Why don't we need to run as priviledged ? can we interact with the socket correctly ? (docker check working ok ?).
(I do like that we explicitely say `"mode": "RO"`).

Reply:
Hm.. docker check does seem to be working, I don't think we'd need privileged for that?
```
    {
      "metric": "docker.uptime",
      "points": [
        [
          1602814475,
          174
        ]
      ],
      "tags": [
        "container_id:7d7c3a0ce2077259d4a60c6910bfccc4ba78af197cc15185dd4a8074ac643643",
        "container_name:mesos-668b5ec2-47ed-42d5-8aeb-fe937c425418",
        "docker_image:datadog/agent-dev:dcos-auth-token-py3",
        "image_name:datadog/agent-dev",
        "image_tag:dcos-auth-token-py3",
        "marathon_app:/datadog-agent",
        "mesos_task:datadog-agent.instance-e4e8ed2e-0d7f-11eb-8e5d-a2cf47f4afdc._app.3",
        "short_image:agent-dev"
      ],
      "host": "i-02274f24c2f929bdc",
      "type": "gauge",
      "interval": 0,
      "source_type_name": "System"
    },
```
But let me confirm and review the docs, I believe we copy-pasted this JSON from original docs and then altered.. and then altered some more.